src/Symfony/Bundle/FrameworkBundle/Test/WebTestCase.php

```diff
@@ -21,6 +21,15 @@
  */
 abstract class WebTestCase extends KernelTestCase
 {
+    use WebTestAssertions;
+
+    protected function tearDown()
```

Comment:
this will cause issues with PHPUnit 8, while KernelTestCase was adapted to support it.

Reply:
What would you recommend then?
Would it be ok to create another trait, like `RemoveClientOnShutdownTrait` or similar?

Reply:
see #30474

Reply:
Awesome! Thanks for the link, I'll do this 👍

Reply:
Fixed in 86fdbb8